core/drivers/stm32_i2c.c

```diff
@@ -1553,3 +1593,97 @@ void stm32_i2c_suspend(struct i2c_handle_s *hi2c)
 
 	hi2c->i2c_state = I2C_STATE_SUSPENDED;
 }
+
+static TEE_Result stm32_get_i2c_dev(struct dt_pargs *args, void *data,
+				    struct i2c_dev **out_device)
+{
+	struct stm32_i2c_dev *stm32_i2c_dev = NULL;
+	paddr_t addr = 0;
+
+	addr = fdt_reg_base_address(args->fdt, args->phandle_node);
+	if (addr == DT_INFO_INVALID_REG) {
+		DMSG("Can't get device I2C address");
+		return TEE_ERROR_GENERIC;
```

Comment:
Is there a more specific error code we could use ? 
Maybe TEE_ERROR_ITEM_NOT_FOUND ?

Reply:
Here it is the DT that does not provide expected information. I think `TEE_ERROR_GENERIC` better applies.

Reply:
Ok seems coherent.